src/Listeners/AddApiAttributes.php

```diff
@@ -22,5 +26,36 @@ public function subscribe(Dispatcher $events)
     public function addAdminAttributes(PrepareUnserializedSettings $event)
     {
         $event->settings['flagrow.bazaar.flagrow-host'] = FlagrowApi::getFlagrowHost();
+        $event->settings['flagrow.bazaar.php.memory_limit-met'] = $this->memoryLimitMet();
+        $event->settings['flagrow.bazaar.php.memory_limit'] = ini_get('memory_limit');
+        $event->settings['flagrow.bazaar.php.memory_requested'] = PackageManager::MEMORY_REQUIRED;
```

Comment:
Why one is "required" and the other "requested", shouldn't we choose a single term ?